src/date/timezone-region-format.js

```diff
@@ -0,0 +1,30 @@
+define([], function() {
+
+/**
+ * regionFormat( date, format, timeZoneId, formatNumber )
+ *
+ * Return date's timezone names according to the format passed.
+ * Eg for format when timezone offset is 180:
+ * "regionFormat": "{0} Time",
+ * "regionFormat-type-standard": "{0} Standard Time",
+ * "regionFormat-type-daylight": "{0} Daylight Time",
+ * "fallbackFormat": "{1} ({0})",
+ */
+return function( date, regionFormatStandard, regionFormatDaylight, standardTzName, daylightTzName,
+exemplarCity ) {
+
+	// var isDST = date.isDST(); FIXME
+	var isDST = false,
+		format;
+	if ( isDST ) {
+		format = ( daylightTzName ) ? daylightTzName :
+		( regionFormatStandard ) ? regionFormatDaylight.replace( /\{0\}/, exemplarCity ) :
+		undefined;
+	} else {
+		format = ( standardTzName ) ? standardTzName :
+		( regionFormatStandard ) ? regionFormatStandard.replace( /\{0\}/, exemplarCity ) :
+		undefined;
+	}
```

Comment:
Other points:
- The parentheses on `( standardTzName ) ?` is unnecessary;
- Why the `undefined` case?